<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GestioneRichieste</class>
 <widget class="QMainWindow" name="GestioneRichieste">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestione Richieste</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnRichiesteEsci">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>550</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Esci</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="grpRicercaCittadino">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>611</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Ricerca Cittadino</string>
    </property>
    <widget class="QPushButton" name="BtnCerca">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>130</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Cerca</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtVia">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblCognome">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Cognome</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblNome">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>70</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nome</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblVia">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Via/Viale</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtCivico">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>100</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblCivico">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>100</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>NÂ°</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtCognome">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtNome">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>70</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblRagioneSociale">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Ragione sociale</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtRagioneSociale">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="rBtnPrivato">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>30</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Privato</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rBtnAzienda">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>30</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Azienda</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableView" name="tblRicercaCittadino">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>781</width>
      <height>111</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="gridStyle">
     <enum>Qt::SolidLine</enum>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QTableView" name="tblRicercaRichieste">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>781</width>
      <height>111</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="gridStyle">
     <enum>Qt::SolidLine</enum>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QLabel" name="lblElencoRichieste">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Elenco richieste :</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
